index.html

```diff
@@ -0,0 +1,249 @@
+            <li><a href=""><img src="shapes/twitter.png" height="20px" height="20px" alt="twitter"></a></li>
+            <li><a href="">My page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </div>
+      </div>
+
+    </div>
+
+    <div class="shapes">
+      <img src="shapes/shape5.png" alt="" class="shape1">
+      <img src="shapes/shape2.png" alt="" class="shape2">
+    </div>
+    <div class="desktop-color">
+      <div class="container">
+        <nav id="navbar">
+          <div class="logo">
+            <a href="index.html" class="logo-link flex">
+              <div class="images flex">
+                <img src="shapes/graphic.png" alt="web development logo">
+                <h1>Dev</h1>
+              </div>
+            </a>
+          </div>
+          <ul class="menu-items flex" id="menu-mobile">
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Contact</a></li>
+            <li><a href="index.html">New</a></li>
+            <li class="signup-btn"><a href="index.html">Signup</a></li>
+          </ul>
+          <div class="btn">
+            <button id="hamburger-btn">
+              <img class="flex" src="shapes/menu1.png" alt="hamburger button">
+            </button>
+            <button id="close-btn">
+              &times
+            </button>
+          </div>
+        </nav>
+      </div>
+
+    </div>
+
+  </header>
+
+  <main>
+    <div class="container">
+      <div class="hero-section flex-column">
+        <div class="shapes">
+          <img src="shapes/shape7.png" alt="hexagon" class="shape7">
+          <img src="shapes/why-shape-dot.png" alt="shapes dots" class="shape-dot">
+          <img src="shapes/hexagon.png" alt="" class="hexagon">
+        </div>
+        <div class="hero-text flex-column">
+          <h3 class="hero-first-title flex-column">
+            "Become a Pro Web Developer"
+          </h3>
+          <h1 class="hero-title flex-column">
+            Learn Web Development
+          </h1>
+          <div class="hero-desc flex-column">
+            <p>
+              Get access to 2700+ web development courses taught by top instructors
+              on our online learning platform. Our courses cover topics for
+              beginners to advanced level.
+            </p>
+            <p>
+              Our top instructors come from leading tech companies
+              and offer expertise in HTML, CSS, JavaScript, and full-stack development.
+              Get started on your web development journey today with our courses.
+            </p>
+          </div>
+        </div>
+        <div class="program-time">
+          <h2>2015.04.24 - NOW </h4>
+            <p>Join our comprehensive seven-month online program and learn essential web development skills from our
+              company in Gisenyi, Rwanda.</p>
+        </div>
+
+      </div>
+    </div>
```

Comment:
Well done on this section 👍🏽 but let's see how the Desktop version can be improved, make sure to increase the font-size (here and in some other places as well), adjust the padding and margin to make it look more like the design. Take a look at the differences👇🏽 

## Your design

![Screenshot 2023-04-28 at 03 21 38](https://user-images.githubusercontent.com/45739963/235081612-5e73e6d7-b0cb-4633-a54a-8573d51f714d.png)

## Figma 

![Screenshot 2023-04-28 at 03 22 10](https://user-images.githubusercontent.com/45739963/235081765-21e33f18-7a9d-4332-84f2-8afcaea7ef6a.png)

Reply:
I can see that i did not match the way the design is but now i have fix the font-size that was no match